contracts/test/OpportunityAdapterIntegration.t.sol

```diff
@@ -2,12 +2,12 @@
 pragma solidity ^0.8.13;
 
 import {Test} from "forge-std/Test.sol";
-import "forge-std/console.sol";
 import "openzeppelin-contracts/contracts/proxy/ERC1967/ERC1967Proxy.sol";
 
 import "src/express-relay/Errors.sol";
 import "src/opportunity-adapter/OpportunityAdapter.sol";
 import {OpportunityAdapterFactory} from "src/opportunity-adapter/OpportunityAdapterFactory.sol";
+import {MulticallAdapter, TargetCall as TargetCall, TokenToSend as TokenToSend, TokenAmount as MulticallAdapterTokenAmount, MulticallParams} from "test/multicall-adapter/MulticallAdapter.sol";
```

Comment:
`TargetCall as TargetCall, TokenToSend as TokenToSend` this part can be simplified